src/plot_api/plot_api.js

```diff
@@ -941,6 +941,7 @@ Plotly.redraw = function(gd) {
 Plotly.newPlot = function (gd, data, layout, config) {
     gd = getGraphDiv(gd);
     plots.purge(gd);
+    gd.removeAllListeners();
```

Comment:
![purge it all](https://upload.wikimedia.org/wikipedia/en/2/29/The_Purge_%E2%80%93_Anarchy_Poster.jpg)